app/assets/styles/_modals.scss

```diff
@@ -16,7 +16,7 @@
       border: none;
       &::before {
         content: '';
-        @include helsinki-logo($black);
+        // @include helsinki-logo($black);
```

Comment:
Any reason? 
If it means to change forever (design changed) -> should be removed totally instead of commenting out

Reply:
Removed